src/domain/community/user/userProfilePage/UserProfilePageView.tsx

```
@@ -11,33 +11,36 @@ import { SpaceHostedItem } from '../../../journey/utils/SpaceHostedItem';
-    () =>
-      contributions?.filter(
-        contribution => contribution.roles?.includes(RoleType.Lead) || contribution.roles?.includes(RoleType.Admin)
-      ) || [],
-    [contributions]
-  );
+  const [filteredMemberships, remainingMemberships] = useFilteredMemberships(contributions, [
+    RoleType.Lead,
+    RoleType.Admin,
+  ]);
+
+  const hasAccountResources = accountResources && accountResources.spaces && accountResources.spaces.length > 0;
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)